src/i18n/locale/en.json

```
@@ -131,7 +142,7 @@
   "components.PersonDetails.nobiography": "No biography available.",
   "components.PlexLoginButton.loading": "Loading…",
   "components.PlexLoginButton.signingin": "Signing in…",
-  "components.PlexLoginButton.signinwithplex": "Sign In",
+  "components.PlexLoginButton.signinwithplex": "Sign In with Plex",
```

```suggestion
  "components.PlexLoginButton.signinwithplex": "Sign In",
```
This is just the button text, and needs to be reverted to the original string (the accordion option already says "Use your Plex account").

---

Commited, thanks! I originally added the text when I was editing the setup flow, but didn't realize the component was reused on the login screen.

---

@acvigue I'll be going through all of the new strings you've added sometime today :)  I kind of did a lot of work in trying to make all the UI text consistent/standardized.